spec/support/crashing_handler.rb

```diff
@@ -0,0 +1,5 @@
+class CrashingHandler < DispatchRider::Handlers::Base
+  def process(params)
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.